signbank/dictionary/models.py

```diff
@@ -565,6 +565,8 @@ class Meta:
     videoexample4_translation = models.CharField(_("Example video 4 English translation"),
                                                  max_length=255, null=True, blank=True, help_text="English translation for video example 4")
 
+
+    fingerspelling = models.NullBooleanField(default=False, blank=True, null=True)
```

Comment:
Can you make sure a verbose_name is listed here so that the capitilisation is correct on the admin and advanced gloss views. Can you also confirm that fingerspelling can definitely be `null`, or whether it should always be true or false (you will need to check the Freelex data for this)